lib/ws_servers/api/snapshots/send_all.js

```diff
@@ -14,5 +13,4 @@ module.exports = async (session, ws) => {
   await sendUserInfo(session, ws, rest)
   await sendWallets(session, ws, rest)
   await sendPositions(session, ws, rest)
-  await sendOrders(session, ws, rest)
```

Comment:
how does frontend show the active orders if we don't send from the backend?

Reply:
the FE will not flush the data anymore, so it keeps track of the AOs from both modes

Reply:
UPDATE: we actually need that, but in a different form, I'll move this task to draft again